pkg/cloud/amazon/s3_storage.go

```diff
@@ -70,9 +71,8 @@ const (
 	// KMSRegionParam is the query parameter for the 'region' in every KMS URI.
 	KMSRegionParam = "REGION"
 
-	// AWSRoleArnParam is the query parameter for the 'role ARN' in an AWS
-	// URI, use if credentials are being verified with Assume Role.
-	AWSRoleArnParam = "AWS_ROLE_ARN"
+	// AssumeRoleParam is query parameter for the chain AWS Role ARNs to assume.
+	AssumeRoleParam = "ASSUME_ROLE"
```

Comment:
just sanity checking, we havent backported any of this to 22.1 right, in which case its okay to switch the param name.